src/components/webview/IntuitaPanelProvider.ts

```diff
@@ -34,10 +36,43 @@ Codemod: ${codemodName}
 };
 
 const selectPanelViewProps = (
+	codemodDescriptionProvider: CodemodDescriptionProvider,
 	state: RootState,
 	rootPath: string,
 ): PanelViewProps | null => {
+	const { activeTabId } = state;
+
+	if (activeTabId === TabKind.community) {
```

Comment:
hmm that's always a good question, if we should shield ourselves from the future bugs by making the code more precise... I am always undecided on such issues...

Reply:
@hbjORbj you cannot always shield yourselves from all the changes, because you don't know how the changes would look like, and you need to choose a proper path between not caring about it at all, or shielding yourself from every possible issue. This is not trivial in my opinion.

Reply:
there's always a road in the middle, making reasonable simplifications when necessary